docs/mvc/security/cors-policy.rst

```diff
@@ -1,8 +1,79 @@
-.. include:: /stub-topic.txt
+Specifying a CORS Policy
+========================
 
-|stub-icon| Specifying a CORS Policy
-====================================
+By `Mike Wasson <https://github.com/MikeWasson>`_ 
 
-.. include:: /stub-notice.txt
+Browser security prevents a web page from making AJAX requests to another domain. This restriction is called the *same-origin policy*, and prevents a malicious site from reading sensitive data from another site. However, sometimes you might want to let other sites make cross-origin requests to your web app.
 
-.. _issue: https://github.com/aspnet/Docs/issues/136
\ No newline at end of file
+`Cross Origin Resource Sharing <http://www.w3.org/TR/cors/>`_ is a W3C standard that allows a server to relax the same-origin policy. Using CORS, a server can explicitly allow some cross-origin requests while rejecting others. This topic shows how to enable CORS in your ASP.NET MVC 6 application. (For background on CORS, see :ref:`cors-how-cors-works`.)
+
+
+Add the CORS package
+--------------------
+
+In your project.json file, add the following:
+
+.. literalinclude:: cors-policy/sample/project.json
+    :language: json
+    :lines: 5,9-11
+    :emphasize-lines: 3
+
+
+Configure CORS
+--------------
+
+To configure CORS, call ``ConfigureCors`` in the ``ConfigureServices`` method of your ``Startup`` class, as shown here:
+
+.. literalinclude:: cors-policy/sample/Startup.cs
```

Comment:
This one points to the sample which uses both poilicy using  ```ConfigureCors``` and using a filter. We should probably show the simplest case here ( using the policy with ```EnableCors``` instead of the filter.